backend/src/controller/auctions.ts

```diff
@@ -247,3 +260,34 @@ export const deleteAuction = async (req: Request, res: Response) => {
         res.status(404).json({ message: error.message });
     }
 };
+
+export const addRemoveLike = async (req: Request, res: Response) => {
+    const { auctionID } = req.params;
+    const { profileID } = req.body;
+
+    const checkIfExistingLike = (likesArray: any) => {
+        return likesArray.findIndex((item: any) => item.like.profileID == profileID);
+    };
+
+    const newLike = {
```

Comment:
question: wouldn't it be enough to just keep an array of profileIDs instead of objects?